<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cmbCAN1Baudrate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MesloLGL Nerd Font Mono</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>USB Connection Settings</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkCAN1Sta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier Prime</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>If youtick this, CAN 1 will be enabled for communication</string>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUSBStatus">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>UNKNOWN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtLog">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>631</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Euphemia</family>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnectUSB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDetect">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click here to detect all the USB serial port connections to be detected and listed in dropdown box&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Detect Devices</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Goudy Old Style</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CAN Controller 1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbSerialPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>78</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDisconnectUSB">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbCAN2Baudrate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInitCAN">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Initialise CAN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Goudy Old Style</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CAN Controller 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkCAN2Sta">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier Prime</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>If youtick this, CAN 1 will be enabled for communication</string>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MesloLGL Nerd Font Mono</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CAN Connection Settings</string>
    </property>
   </widget>
   <zorder>cmbCAN1Baudrate</zorder>
   <zorder>label_3</zorder>
   <zorder>chkCAN1Sta</zorder>
   <zorder>txtLog</zorder>
   <zorder>btnConnectUSB</zorder>
   <zorder>btnDetect</zorder>
   <zorder>label_4</zorder>
   <zorder>cmbSerialPort</zorder>
   <zorder>btnDisconnectUSB</zorder>
   <zorder>cmbCAN2Baudrate</zorder>
   <zorder>btnInitCAN</zorder>
   <zorder>label_6</zorder>
   <zorder>chkCAN2Sta</zorder>
   <zorder>label_5</zorder>
   <zorder>lblUSBStatus</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1283</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPanes">
    <property name="title">
     <string>Panes</string>
    </property>
    <addaction name="actionFrame_Generator"/>
    <addaction name="actionFrame_Transaction"/>
    <addaction name="actionMultiple_CAN_Frame_Handler"/>
   </widget>
   <addaction name="menuPanes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFrame_Generator">
   <property name="text">
    <string>Frame Generator</string>
   </property>
  </action>
  <action name="actionFrame_Transaction">
   <property name="text">
    <string>Frame Transaction</string>
   </property>
  </action>
  <action name="actionMultiple_CAN_Frame_Handler">
   <property name="text">
    <string>Multiple CAN Frame Handler</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
